org.opent2t.sample.binaryswitch.superpopular/com.insteon.binaryswitch/js/tests/unitTests.js

```diff
@@ -6,7 +6,7 @@ var MockHub = require('opent2t-device-insteonhub/mockInsteonHub');
 var mockHub = new MockHub(deviceData);
 
 var settings = {
-    createTranslator: mockHub.createTranslator(translatorPath, deviceData.base_state.data.object_id),
```

Comment:
Thanks for fixing these -- i took a look and these were in bad shape last evening :( appreciate it!  While you are at it, because there aer so many different test files, could you update the readme's on how to execute and where to extend if we need to ?  Also a note to update teh mock implementation if there is a behavior change to the real translators (Eg error handling)